### PR TITLE
[IMP] mail: guard ActivityMarkDonePopover

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2797754

### CODE REVIEW

File: addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.xml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ActivityMarkDonePopover" owl="1">
-        <div class="o_ActivityMarkDonePopover" t-attf-class="{{ className }}" t-on-keydown="activityMarkDonePopoverView and activityMarkDonePopoverView.onKeydown" t-ref="root">
-            <t t-if="activityMarkDonePopoverView">
+        <t t-if="activityMarkDonePopoverView">
+            <div class="o_ActivityMarkDonePopover" t-attf-class="{{ className }}" t-on-keydown="activityMarkDonePopoverView and activityMarkDonePopoverView.onKeydown" t-ref="root">
```
Comment (oomsveta) [inline addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.xml:6]: is `activityMarkDonePopoverView and` still needed now that this node is guarded?

Comment (seb-odoo) [reply addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.xml:6]: Good catch, note that it's the main reason to do this change.

Owl 2 does not support `undefined` event handler, so previous guard is useless.

Comment (alexkuhn) [reply addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.xml]: Changes done

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**